.github/workflows/lint-and-test.yml

```diff
@@ -83,6 +83,7 @@ jobs:
       - read-playwright-version
     container:
       image: mcr.microsoft.com/playwright:${{ needs.read-playwright-version.outputs.version }}
+      options: --user 1001
```

Comment:
The repo is checked out under a different user to the container by default, meaning subdirectories can't be written to by the container without a workaround like this.